open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/utils/ConsentRetrievalUtil.java

```diff
@@ -190,16 +187,18 @@ public static void populateDebtorAccount(JSONObject initiation, JSONArray consen
             jsonElementDebtor.appendField(ConsentExtensionConstants.DATA_SIMPLE, debtorAccountArray);
             consentDataJSON.add(jsonElementDebtor);
         }
+
```

Comment:
Shall we remove these additional new lines? check other places as well.

Reply:
Fixed in [60a3cf3]